app/models/light/user.rb

```diff
@@ -81,12 +84,12 @@ def self.import(file, email='')
     end
 
     def self.users_for_opt_in_mail
-      date = Date.today.strftime("%Y%m")
+      date = Date.today.strftime('%Y%m')
       self.new_users.where(:sent_on.nin => [date], is_blocked: false).order_by([:email_id, :asc])
     end
 
     def self.users_for_opt_out_mail
-      date = Date.today.strftime("%Y%m")
+      date = Date.today.strftime('%Y%m')
```

Comment:
Both methods users_for_opt_out_mail and users_for_opt_in_mail  looks same. Use single method if the condition is same

Reply:
Okay, Will update that